### DISCUSSION TITLE
What does PARK_FILE_MIN_VERSION do?

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#16537] [Q&A]
Right now both of them are set to `0x8`, but there is code for importing parkfile versions less than `0x4` (updating to use the track draw feature). What does PARK_FILE_MIN_VERSION do?

**Comment by duncanspumpkin** [comment]
I've stated a few times that it's value is confusing and it should be removed as ultimately people only care about the minimum version that is backwards compatible.

**Comment by IntelOrca** [comment]
If the park file is changed to add or write new data at the end of a struct. The min version does not need to change, because older versions of OpenRCT2 will be able to open it. But new versions will be able to load the new data, based on the version field.

**Reply by spacek531** [reply]
What happens to data that older versions do not recognize, are they preserved or discarded on import and export?

**Reply by IntelOrca** [reply]
Discarded, we still need to make a dialog that opens up when loading a park created with a newer version, showing a warning that this will happen.

**Comment by spacek531** [comment]
Is there any place this is utilized? I can't see it used all that often

**Reply by duncanspumpkin** [reply]
No, it is not used at present as we have never had a situation where it has made sense to use it.

**Comment by spacek531** [comment]
The game will import parks with newer versions than an exported park's min version, which seems like it is intended. If min version was not intended to stop importing of newer versions, I question why it exists in the first place.
